examples/src/http_server_html_template/views/index.rs.html

```diff
@@ -0,0 +1,21 @@
+
+@()
```

Comment:
do we need it to be empty?

Reply:
Yes, no parameters from rust to html in this simple example